source/mainnet/smart-contracts/guides/build-schema.rst

```diff
@@ -180,3 +180,36 @@ where ``FILE`` is a path of the file to create:
 .. code-block:: console
 
    $cargo concordium build --schema-out="/some/path/schema.bin"
+
+If using ``cargo concordium`` version 2.6.0 or newer then the schema can be
+output in JSON format that can be more suitable for use in dApps. When building
+the contract use ``--schema-json-out=DIR`` to output the schema for each
+contract in the module to a JSON file inside the directory ``DIR``. The
+directory must exist.
+
+.. code-block:: console
+
+   $cargo concordium build --schema-json-out="/some/path"
+
+Both ``--schema-out`` and ``--schema-json-out`` can be used at the same time and
+schemas in both formats will be output.
+
+.. code-block:: console
+
+   $cargo concordium build --schema-out="/some/path/schema.bin" --schema-json-out="/some/path"
+
+Converting a binary schema to JSON
+----------------------------------
+
+To convert an existing binary schema (obtained via ``--schema-out``) use the
+``cargo concordium schema-json`` subcommand, e.g.,
+
+.. code-block:: console
+
+   $cargo concordium schema-json --schema schema/schema.bin --out="/some/path"
```

Comment:
Suggestion: we could make all commands consistent, either

cargo concordium schema-json --schema schema/schema.bin --out ./some/path

or 

cargo concordium schema-json --schema="schema/schema.bin" --out="./some/path"

Reply:
Good point.